platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -391,4 +425,35 @@ private ConfigurationMigrationManager createConfigurationMigrationManager() {
 
         return migrationWarnings;
     }
+
+    private void verifyDescriptionEqual(Describable describable, DescribableBean bean) {
+        assert (describable.getId()
+                .equals(bean.getId()));
+        assert (describable.getTitle()
+                .equals(bean.getTitle()));
+        assert (describable.getDescription()
+                .equals(bean.getDescription()));
+        assert (describable.getOrganization()
+                .equals(bean.getOrganization()));
+        assert (describable.getVersion()
+                .equals(bean.getVersion()));
+    }
+}
+
+class TestMigratable extends DescribableBean implements DataMigratable {
```

Comment:
After talking to @Lambeaux, this test can be left alone for now as it does provide some value. It could however be reworked and moved out of this class in the future.